pkgs/development/compilers/halide/default.nix

```diff
@@ -84,6 +87,7 @@ stdenv.mkDerivation rec {
     libjpeg
     eigen
     openblas
+    flatbuffers
```

Comment:
I think flatbuffers is a code generator, in which case it should be in `nativeBuildInputs` instead. You can build with `strictDeps=true;` to be sure. (you can leave `strictDeps` in the package, that's a good thing)

Reply:
Thanks @symphorien. Moved flatbuffers to `nativeBuildInputs` and set `strictDeps=true`. Tests pass for me locally.